pgctl/cli.py

```diff
@@ -457,6 +494,22 @@ def parser():
     return parser
 
 
+def _humanize_seconds(seconds):
+    for period_name, period_length in (
```

Comment:
Yeah, I feel this too. That's actually what I did first, but then switched it to this for a couple reasons:
* It took up a large amount of space
* I wasn't really sure there was value in the exact amount of time. I think normally people just want a ballpark time it's been up (e.g. has it been restarted recently?). I'm not sure if there's really a case where somebody cares down to the second if it's already on the order of magnitude of days. But I'm not too crazy about it this way, either.

I'll put it back to that for now and see if anyone has a strong opinion either way.